node/core/dispute-coordinator/src/real/ordering/mod.rs

```
@@ -171,22 +179,36 @@ impl OrderingProvider {
 		update: &ActiveLeavesUpdate,
 	) -> Result<()> {
 		if let Some(activated) = update.activated.as_ref() {
-			// Get included events:
-			let included = get_candidate_events(sender, activated.hash)
-				.await?
-				.into_iter()
-				.filter_map(|ev| match ev {
-					CandidateEvent::CandidateIncluded(receipt, _, _, _) => Some(receipt),
-					_ => None,
-				});
-			for receipt in included {
-				let candidate_hash = receipt.hash();
-				self.included_candidates.insert(candidate_hash);
-				self.candidates_by_block_number
-					.entry(activated.number)
-					.or_default()
-					.insert(candidate_hash);
+			// Fetch ancestors of the activated leaf.
+			let ancestors = self
+				.get_block_ancestors(sender, activated.hash, activated.number)
+				.await
+				.unwrap_or_default();
```

We should not silently swallow errors, we should log them - debug level.